static/sass/_snapcraft_patterns_overlay.scss

```diff
@@ -0,0 +1,25 @@
+@mixin snapcraft-p-overlay {
+  .p-overlay {
```

Comment:
There is already a Vanilla modal pattern (that we use on snap details page): https://docs.vanillaframework.io/en/patterns/modal

Is there any reason to create this as separate pattern?

Reply:
"Added a new simplified "p-overlay" pattern (to be rolled out to other pages (snap details))" from description. But this is removed now so not required.